metricflow-semantics/pyproject.toml

```diff
@@ -58,5 +58,12 @@ features = [
 [tool.hatch.build.targets.sdist]
 packages = ["metricflow_semantics"]
 
+# include doesn't seem to work with packages, so using force include.
+[tool.hatch.build.targets.sdist.force-include]
+"ATTRIBUTION.md" = "ATTRIBUTION.md"
```

Comment:
Huh. I think this package-local attribution file is the correct layout but I just realized I probably need to generate attribution files for each package independently, since their dependencies are different.